props_lab/src/components/donationDisplay.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const DonationDisplay = ({ name, amount, caption }) => {
+    return(
+        <div className={"donationDisplayContainer"}>
```

Comment:
If its just going to be a string it doesnt have to be in the braces. You use braces when you want whats inside to be thought of as JSX